test/controllers/update/update.review.statement.controller.spec.ts

```diff
@@ -156,7 +157,9 @@ describe("Update review overseas entity information controller tests", () => {
       mockIsActiveFeature.mockReturnValueOnce(true);
       mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_UPDATE_BO_MOCK);
       mockPaymentsSession.mockReturnValueOnce(PAYMENT_LINK_JOURNEY);
-      const resp = await request(app).post(UPDATE_REVIEW_STATEMENT_URL);
+      const resp = await request(app)
+        .post(UPDATE_REVIEW_STATEMENT_URL)
+        .send({ no_change_review_statement: '1' });
```

Comment:
The test name mentions 'no_change_review_statement' being undefined, so we either need to keep the test as is and potentially update the logic in the controller or update the name of the test?

Reply:
Changed name (its outdated now), cheers!